### ISSUE TITLE
Change "http://" namespace urls to use "https://"

### ISSUE THREAD

**Issue by dgeorge** [adobe/xdm#195]
In the comments for https://github.com/adobe/xdm/issues/188, @trieloff posted a complete list of all namespace URLs referenced in XDM schemas.

The list includes the following XDM URLs, which begin with  "http://" instead of "https://"

```
http://ns.adobe.com/xdm/IMSOrg
http://ns.adobe.com/xdm/IMSUser
http://ns.adobe.com/xdm/aem/instance
http://ns.adobe.com/xdm/aem/user
http://ns.adobe.com/xdm/asset
http://ns.adobe.com/xdm/assets/asset
http://ns.adobe.com/xdm/assets/asset#
http://ns.adobe.com/xdm/event/created
http://ns.adobe.com/xdm/event/published
http://ns.adobe.com/xdm/event/unpublished
http://ns.adobe.com/xdm/event/updated
http://ns.adobe.com/xdm/page
http://ns.adobe.com/xdm/services
```

I lack all of the context behind those URL choices.  On the surface, though, it would seem that they should be changed to use "https://".  Also, the following already-existing URLs may be redundant with entries in the above list:

```
https://ns.adobe.com/xdm-extensions/ims/organization
https://ns.adobe.com/xdm-extensions/ims/user OR https://ns.adobe.com/xdm/base/imsuser#
https://ns.adobe.com/xdm/aem/user
https://ns.adobe.com/xdm/assets/asset
https://ns.adobe.com/xdm/assets/asset#/definitions/asset
https://ns.adobe.com/xdm/common/event/created
https://ns.adobe.com/xdm/common/event/published
https://ns.adobe.com/xdm/common/event/updated
https://ns.adobe.com/xdm/common/page OR https://ns.adobe.com/xdm/content/page-component
```

**Comment by trieloff** [comment]
@dgeorge thanks for having a look at this.

**Comment by dgeorge** [comment]
@asteed If you'd prefer to refactor this into multiple smaller issues, that's totally fine with me.

**Comment by trieloff** [comment]
Yes, it's needed for 0.9, as it is a breaking change.

**Comment by asteed** [comment]
The following is a list of changes made to the items raised by @dgeorge 

| Original | Updated |
| --- | --- |
| `http://ns.adobe.com/xdm/IMSOrg` | `https://ns.adobe.com/xdm-extensions/ims/organization` |
| `http://ns.adobe.com/xdm/IMSUser` | `https://ns.adobe.com/xdm-extensions/ims/user` |
| `http://ns.adobe.com/xdm/aem/instance` | `https://ns.adobe.com/xdm/content/repository` |
| `http://ns.adobe.com/xdm/aem/user` | `https://ns.adobe.com/xdm-extensions/aem/user` |
| `http://ns.adobe.com/xdm/asset` | *No longer found in the repository* |
| `http://ns.adobe.com/xdm/assets/asset` | `http://ns.adobe.com/xdm/assets/asset` |
| `http://ns.adobe.com/xdm/assets/asset#` | `https://ns.adobe.com/xdm/assets/asset` |
| `http://ns.adobe.com/xdm/event/created` | `https://ns.adobe.com/xdm/common/event/created` |
| `http://ns.adobe.com/xdm/event/published` | `https://ns.adobe.com/xdm/common/event/published` |
| `http://ns.adobe.com/xdm/event/unpublished` | `https://ns.adobe.com/xdm/common/event/unpublished` |
| `http://ns.adobe.com/xdm/event/updated` | `https://ns.adobe.com/xdm/common/event/updated` |
| `http://ns.adobe.com/xdm/page` | `https://ns.adobe.com/xdm/content/componentized-page` |
| `http://ns.adobe.com/xdm/services` | `https://ns.adobe.com/xdm/content/repository` |
| `https://ns.adobe.com/xdm/base/imsuser#` | `https://ns.adobe.com/xdm-extensions/ims/user` |
| `https://ns.adobe.com/xdm/aem/user` | `https://ns.adobe.com/xdm-extensions/aem/user` |
| `https://ns.adobe.com/xdm/assets/asset` | *Kept as-is* |
| `https://ns.adobe.com/xdm/assets/asset#/definitions/asset` | *Kept as-is* |

@trieloff These will be addressed once #201 is merged which contained the changes to a large number of the above items.

**Comment by lrosenthol** [comment]
These changes look fine to me.  @trieloff - can you review and then close (assuming no issues)
